docs/new-architecture-app-renderer-android.md

```diff
@@ -116,16 +118,18 @@ First, make sure you followed the instructions to [Enabling the New Renderer (Fa
 
 ```javascript
 import codegenNativeComponent from 'react-native/Libraries/Utilities/codegenNativeComponent';
+
 // babel-plugin-codegen will replace the function call to use NativeComponentRegistry
 // 'RCTWebView' is interopped by RCTFabricComponentsPlugins
-export default (codegenNativeComponent<NativeProps>(
-  'RCTWebView',
-): HostComponent<NativeProps>);
+
+export default (codegenNativeComponent <
+  NativeProps >
+  'RCTWebView': HostComponent<NativeProps>);
 ```
 
 4. **[Flow users]** Make sure your native component has Flow types for its props, since the JS codegen uses these types to generate the type-safe native implementation of the component. The codegen generates C++ classes during the build time, which guarantees that the native implementation is always up-to-date with its JS interface. Use [these c++ compatible types](https://github.com/facebook/react-native/blob/main/Libraries/Types/CodegenTypes.js#L28-L30).
 
-```javascript title="RNTMyNativeViewNativeComponent.js"
+```ts title="RNTMyNativeViewNativeComponent.js"
```

Comment:
Quoting me from above:

> Currently the Flow code is unhighlighted [...] It also looks like the TypeScript highlight works a bit better than JS one in the Flow files [...]

However I'm going to add soon the Flow highlighting, I have a separate PR with this and Webpack improvements, because I want to land it on `main` first.

Reply:
Yup sorry my bad 👍 closing here